packages/lesswrong/components/community/modules/CommunityMembers.tsx

```diff
@@ -23,7 +23,7 @@ const styles = createStyles((theme: ThemeType): JssStyles => ({
     '@media (max-width: 1200px)': {
       padding: '0 20px',
     },
-    [theme.breakpoints.down('sm')]: {
+    [theme.breakpoints.down('xs')]: {
```

Comment:
I made it so that the map appears on some smaller screen sizes, since I felt it worked ok and the page is a lot better with a map